CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.8.1]
+
+### Added
+
+- Added p and z-linux images to Docker Hub ([#332](https://github.com/application-stacks/runtime-component-operator/pull/332))
+
```

Comment:
We should add a Fixed section with the fix for image with 3 or more slashes issue